motioneye/templates/base.html

```diff
@@ -4,7 +4,7 @@
         {% block meta %}
             <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
             <meta http-equiv="X-UA-Compatible" content="IE=Edge">
-            <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=15.0, user-scalable=yes">
```

Comment:
```suggestion
            <meta name="viewport" content="width=device-width, initial-scale=1">
```
Rust removing this attribute should allow unlimited zooming. An initial scale makes sense, but I agree that preventing zooming on mobile devices is nearly always a bad idea, since users usually attempt it for reasons.

Reply:
Okay, the default `maximum-scale` seems to be usually lower than `15.0` and depends on the browser, e.g. `5.0` on iOS. But I think it is very rare that such deep zoom is required and we should simply stick with the default, aside of the reasonable initial scale.